novawallet/Modules/Swaps/Setup/SwapSetupViewController.swift

```diff
@@ -200,7 +209,8 @@ extension SwapSetupViewController: SwapSetupViewProtocol {
         switch viewModel {
         case let .asset(assetViewModel):
             rootView.payAmountInputView.bind(assetViewModel: assetViewModel)
-            rootView.depositTokenButton.imageWithTitleView?.title = "Get \(assetViewModel.symbol)"
+            depositTokenSymbol = assetViewModel.symbol
```

Comment:
Thank for fixing this. I think we just need to apply current localisation without saving symbol locally. In case localisation change presenter will provide new view model.